spec/unit/lib/cloud_controller/runner_spec.rb

```diff
@@ -120,17 +158,15 @@ module VCAP::CloudController
         allow(StenoConfigurer).to receive(:new).and_return(steno_configurer)
         allow(steno_configurer).to receive(:configure)
 
-        subject.run!
-        subject.run!
+        subject
 
         expect(steno_configurer).to have_received(:configure).once
       end
 
       it 'sets up telemetry logging once' do
         allow(TelemetryLogger).to receive(:init)
 
-        subject.run!
-        subject.run!
+        subject
```

Comment:
Good point. I thought with `let` I could pull out the initialisation of the runner out of the test context, so that if I would test for something which happens in the constructor, the test would fail. Unfortunately that does not work. I changed it back to use `subject.run!` again.